server/database/queries/getHashPassword.js

```diff
@@ -0,0 +1,5 @@
+const connection = require('../connection');
+
+const getHashPassword = (email) => connection.query(`SELECT pasword FROM users WHERE email = '${email}';`);
```

Comment:
Again, as what Lina  mentioned to you [here](https://github.com/GSG-G10/Crown-Ecommerce/pull/33#discussion_r721141788)